Sources/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -67,7 +67,10 @@ class PurchasesOrchestrator {
     private let lock = NSRecursiveLock()
 
     @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
-    lazy var storeKit2Listener = StoreKit2TransactionListener(delegate: self)
+    lazy var storeKit2TransactionListener = StoreKit2TransactionListener(delegate: self)
```

Comment:
In the future, we could create a single listener that would be composed of other "sublisteners"

Reply:
Can this be `private(set)`?

Reply:
That's fine then!

Reply:
That's fine then!